org.mwc.debrief.ui_test/test-cases/analyzing_Sensor_Data/format_sensor_data.test

```diff
@@ -96,7 +98,7 @@ with [get-view Properties | get-tree] {
 }
 with [get-view Outline | get-tree] {
     get-item -path "Track:Frigate/Sensors \\(5 items\\)" | expand
-    get-cell 5 0 | click
+    select  "Track:Frigate/Sensors \\(5 items\\)/Sensor:Frigate_Optic2 \\(16 cuts\\)"
```

Comment:
@helenayele - could you take a note of this change?  If/when we develop more RCPTT tests, it's fragile to navigate by a command like `get-cell 5 0`, since adding any data may break the test.  It's more robust to direct RCPTT to an element by naming it.